src/hotspot/cpu/riscv/riscv.ad

```diff
@@ -10066,6 +10069,23 @@ instruct CallLeafDirect(method meth, rFlagsReg cr)
   ins_pipe(pipe_class_call);
 %}
 
+// Call Runtime Instruction without safepoint and with vector arguments
+
+instruct CallLeafDirectVector(method meth)
+%{
+  match(CallLeafVector);
+
+  effect(USE meth);
```

Comment:
It's possible for the runtime call to clobber `cr` (aka `t1`). So safer to add `KILL cr` to the effect.

Reply:
Good, it will definitely be clobbered.

Reply:
Thanks for catching!